util/util.go

```diff
@@ -0,0 +1,50 @@
+package util
+
+import (
+	"net/http"
+
+	"github.com/labstack/echo"
+)
+
+//Response struct
+type Response struct {
+	Status string      `json:"status"`
+	Data   interface{} `json:"data"`
+}
+
+//ErrorBody struct
+type ErrorBody struct {
+	Code    string `json:"code"`
+	Message string `json:"message"`
+	Status  string `json:"status"`
+}
+
+//ErrorResponse struct
+type ErrorResponse struct {
+	Error ErrorBody `json:"error"`
+}
+
+//SendSuccess utility function
+func SendSuccess(c echo.Context, data interface{}) error {
```

Comment:
I think all this could have been abstracted into a sendMessage function

Everything could have been sent via data interface and use the status as needed